datastore/gcloud/aio/datastore/constants.py

```diff
@@ -61,23 +61,14 @@ class TypeName(enum.Enum):
     BLOB = 'blobValue'
     BOOLEAN = 'booleanValue'
     DOUBLE = 'doubleValue'
+    ENTITY = 'entityValue'
     INTEGER = 'integerValue'
+    KEY = 'keyValue'
     NULL = 'nullValue'
     STRING = 'stringValue'
     TIMESTAMP = 'timestampValue'
 
 
-# TODO: support more than just scalars
```

Comment:
I moved this to utils since some items are added dynamically now

Reply:
See other comment -- you'll probably need to keep this as a comment and have the dynamic modifications stored separately.